.github/scripts/assign_reviewers.py

```diff
@@ -54,6 +54,21 @@ def get_file_owners(file_path, codeowners_lines):
             return owners  # Remember, can still be empty!
     return []  # Should never happen, but just in case
 
+def pr_author_is_in_hf(pr_author, codeowners_lines):
+    # Check if the PR author is in the codeowners file
+    for line in codeowners_lines:
+        line = line.split('#')[0].strip()
+        if not line:
+            continue
+
+        # Split into pattern and owners
+        parts = line.split()
+        owners = [owner.removeprefix("@") for owner in parts[1:]]
+
+        if pr_author in owners:
+            return True
+    return False
```

Comment:
could we simply using a regex with f'@{pr_autho}' against the codeowner file content? It's justa nit so up to you, but potentially a one line code.

Reply:
It'd probably work, but it feels just the tiniest bit fragile to me, like it'll be harder to change if we ever change any file formats or anything! You're probably right, but I'm going to leave it as-is just because I'm paranoid, lol